api/client/src/intTest/java/org/projectnessie/client/auth/oauth2/ITOAuth2Client.java

```diff
@@ -73,7 +73,7 @@ static void setUpKeycloak() {
         URI.create(KEYCLOAK.getAuthServerUrl() + "/realms/master/protocol/openid-connect/token");
     Keycloak keycloakAdmin = KEYCLOAK.getKeycloakAdminClient();
     master = keycloakAdmin.realms().realm("master");
-    updateMasterRealm(10, 15);
+    updateMasterRealm(5, 10);
```

Comment:
I thought we could keep these changes though, it's nice to have a shorter test execution time.